model/k8s/deprecated/metrics-deprecated.yaml

```diff
@@ -8,10 +8,192 @@ groups:
     stability: development
     deprecated:
       reason: renamed
-      renamed_to: k8s.replicationcontroller.desired_pods
```

Comment:
Should we maybe add `k8s.replicationcontroller.desired_pods` as an additional deprecated metric?
(as this is the second time this metric is being renamed)

Otherwise `k8s.replicationcontroller.desired_pods` wouldn't be mentioned anywhere as a deprecated metric with only the original (from the first rename) name being documented here (i.e. `k8s.replication_controller.desired_pods`).

Reply:
I had both metrics originally, but the validation failed cause it was pointing to a non-existing metric (the first rename).

Reply:
I believe the tooling expects all the previous metric names to remain around.

Given these are not "stable" I think there's less enforcement around keeping things around, although it is risky depending on whether the intermediate name was used.

If you have a caveat asking folks NOT to migrate to new metrics past version "X" then you can drop the intermediary if it's between "X" and now, as (theoretically) anyone taking a dependency there new this might be changing.

Reply:
If I understand correctly:

- metric A was renamed to B
- then metric B was renamed to C

Should deprecation note on A now point to C? 
Yes!

If you migrate from old version with metric A, you should migrate directly to C, no need to even know that there was B in the middle. 
If someone is looking for migration path to older version of semconv (before C existed), they could find B in that version.